CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contribution Guidelines
+
+Follow the [Code of Conduct][1].
```

Comment:
Maybe something like "All contributors are expected to follow the code of conduct."

Then give people a path back to the front door (e.g., "Use the discussion board to ask questions or recommend resources...")

And give some context to the part that's already here ("Request changes to these pages by...")